src/tokens/TokenBalanceItem.tsx

```diff
@@ -0,0 +1,118 @@
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { formatValueToDisplay } from 'src/components/TokenDisplay'
+import TokenIcon from 'src/components/TokenIcon'
+import Touchable from 'src/components/Touchable'
+import { getLocalCurrencySymbol, usdToLocalCurrencyRateSelector } from 'src/localCurrency/selectors'
+import useSelector from 'src/redux/useSelector'
+import colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { TokenBalance } from 'src/tokens/slice'
+
+const TokenBalanceText = (token: TokenBalance) => {
```

Comment:
nit: maybe TokenBalanceInLocalAmount? I'd also suggest exporting this now and have specific tests for this component since we need it in ACT-915, but I can also take it up as part of that issue.